src/amo/pages/UserProfileEdit/index.js

```diff
@@ -207,13 +206,15 @@ export class UserProfileEditBase extends React.Component<Props, State> {
           toPath = `/${toPath}`;
```

Comment:
I am not sure why this is needed but it prevents a potential security issue (when `to` contains an absolute URL).

Reply:
Yes, it forces anything from the `to` param to be a relative URL, which is what we want. There is even a [test for it](https://github.com/mozilla/addons-frontend/blob/13141c121e74be5df3eaf6d64692f22952e12fad/tests/unit/amo/pages/TestUserProfileEdit.js#L936-L940).